src/App.jsx

```diff
@@ -1,26 +1,69 @@
-import { BrowserRouter, Routes, Route, Link } from "react-router-dom"
+import { useState } from 'react'
+import { BrowserRouter, Routes, Route, Link } from 'react-router-dom'
+import { deleteCocktail } from './services/cocktails.service'
 
-// pages
-import Home from "./pages/Home"
-import Create from "./pages/Create"
-import Update from "./pages/Update"
+import Home from './pages/Home'
+import Create from './pages/Create'
+import Update from './pages/Update'
 
+import CocktailModal from './components/CocktailModal'
 
 function App() {
+  const [cocktails, setCocktails] = useState(null)
+  const [showModal, setShowModal] = useState(false)
+  const [currentModalCocktail, setCurrentModalCocktail] = useState(null)
+
+  const handleDelete = async (id) => {
```

Comment:
I think for the scope of the project this is still ok, but probably these functions could go in the `Home` component. Like: The `App` component should be concerned about rendering a `Home` component, but these behaviors might be specific to `Home` so it's better to leave them there instead of passing them from here.

In the `App` component you might be concerned with initializing some state, some provider, translations, etc. Stuff related to specific business logic (how many cocktails do we have to show? should we add a cocktail? should we delete a cocktail? etc) should probably be part of the `Home` component.